Solutions/Tanium/images/Tanium.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 523.12 94.19"><defs><style>.cls-1{fill:#fff;}</style></defs><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path class="cls-1" d="M29,40H93.66A47.1,47.1,0,1,0,47.1,94.19a47.74,47.74,0,0,0,6.58-.47v-39H29Z"/><path class="cls-1" d="M71.22,87.54a47.12,47.12,0,0,0,22.34-32.8H71.22Z"/><path class="cls-1" d="M514.9,76.1h-1.36v-.75h3.58v.75h-1.35v3.69h-.87Zm7.36.2h0L521,79.79h-.67L519,76.3h0v3.49h-.86V75.35h1.39l1.11,3,1.11-3h1.39v4.44h-.86Z"/><polygon class="cls-1" points="111.19 26.36 111.19 39.65 133.43 39.65 133.43 79.79 149.25 79.79 149.25 39.65 171.49 39.65 171.49 26.36 111.19 26.36"/><path class="cls-1" d="M200,38.76l10.26,20.07H190Zm-10.27-12.4L161.77,79.79h17.68l4.63-9.44h32.08l4.91,9.44h17.61L210.31,26.36Z"/><polygon class="cls-1" points="297.28 26.36 297.28 65.18 297.13 65.18 267.41 26.36 242.89 26.36 242.89 79.79 258.71 79.79 258.71 40.98 258.86 40.98 288.58 79.79 313.1 79.79 313.1 26.36 297.28 26.36"/><rect class="cls-1" x="322.37" y="26.36" width="15.82" height="53.43"/><path class="cls-1" d="M347.47,26.36H363.3V57.21c0,6.79,2.35,9.45,8.83,9.45h15.69c6.48,0,8.83-2.66,8.83-9.45V26.36h15.83V60.75c0,13.29-6.63,19.63-21.17,19.63H368.64c-14.54,0-21.17-6.34-21.17-19.63Z"/><polygon class="cls-1" points="421.75 26.36 446.06 26.36 463.95 60.75 481.84 26.36 506.15 26.36 506.15 79.79 490.75 79.79 490.75 41.42 490.61 41.42 469.87 79.79 458.03 79.79 437.29 41.42 437.15 41.42 437.15 79.79 421.75 79.79 421.75 26.36"/></g></g></svg>
```

Comment:
@ian-hands-tanium, can you please update the similar logo added in Logos/Tanium.svg, here as well, thanks.